### DISCUSSION TITLE
Spi based reciewers

### DISCUSSION THREAD

**Discussion by techi9** [ExpressLRS/ExpressLRS#1783] [Q&A]
I just got into hobby and wanted to know is there a way to see elrs version on spi based reciewer (mobula 7 elrs)? And what version of elrs I should flash my remote?

**Comment by SunjunKim** [comment]
with BetaFlight 4.3, ELRS 2.x versions are compatible.
ELRS 3.x will be compatible with the future release of BetaFlight (possibly 4.4)

Unfortunately, the further versions of ELRS will not guarantee compatibility with SPI receiver boards.
